app/redux/ducks/classify.spec.js

```diff
@@ -454,4 +456,106 @@ describe('Classifier actions', function () {
       expect(newState.classification.gold_standard).to.be.undefined;
     });
   });
+
+  describe('load workflow', function () {
+    let awaitWorkflow;
+    let state;
+    let storeState;
+    let initialState = {
+      classification: null,
+      goldStandardMode: false,
+      intervention: null,
+      upcomingSubjects: [],
+      workflow: { id: 'b' }
+    };
+    const fakeDispatch = sinon.stub().callsFake(function (action) {
+      if(typeof action === 'function') {
+        action = action(fakeDispatch);
+      }
+      storeState = reducer(state, action);
```

Comment:
`state` is always the initial state here. The dispatcher should be applying the action to `storeState`.